runtime/onert/core/include/compiler/train/TrainingInfo.h

```diff
@@ -58,7 +58,7 @@ class TrainingInfo
 
 private:
   ir::train::LossInfo _loss_info;
-  ir::train::OptimizerInfo _optimizer_info{ir::train::OptimizerCode::Invalid, 0};
+  ir::train::OptimizerInfo _optimizer_info;
   uint32_t _batch_size = 0;
```

Comment:
#### background

Other `ir` in train (`ir::train::LossInfo`) has a default constructor. 
https://github.com/Samsung/ONE/blob/c86131b565fcf2a61c991fefbd4d4a11715fea3f/runtime/onert/core/include/ir/train/LossInfo.h#L50-L53

So Let's add a default constructor to OptimizerInfo for consistency.